doc/api/fs.md

```diff
@@ -3324,6 +3324,43 @@ a colon, Node.js will open a file system stream, as described by
 Functions based on `fs.open()` exhibit this behavior as well:
 `fs.writeFile()`, `fs.readFile()`, etc.
 
+### `fs.openAsBlob(path)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `path` {string|Buffer|URL}
+* Return: {Promise} containing {Blob}
+
+Returns a {Blob} whose data is backed by the given file.
+
+The file must not be modified after the {Blob} is created. Any modifications
+will cause reading the {Blob} data to fail with an `'ERR_BLOB_READ_FAILED'`
```

Comment:
I have tried this in browser before (doing modification to it) after obtaining a file. the error class i get back is actually: 
```js
new DOMException(
  'The requested file could not be read, typically due to permission problems that have occurred after a reference to a file was acquired.', 
  'NotReadableError'
)
```
thinking that the same error message would be nice to have

Reply:
it was also the error message i used in my [own impl](https://github.com/node-fetch/fetch-blob/blob/57e4daef36081936581d14509b6cc618d87ab9e2/from.js#L136).

```js
    const { mtimeMs, size } = await stat(this.#path)

    if (mtimeMs > this.lastModified || this.originalSize !== size) {
      throw new DOMException('The requested file could not be read, typically due to permission problems that have occurred after a reference to a file was acquired.', 'NotReadableError')
    }
```
(to match what browser throws after being modified)